### ISSUE TITLE
[Bug] TapGestureRecognizer Not working on Path on iOS

### ISSUE THREAD

**Issue by ColeXm** [xamarin/Xamarin.Forms#13020]
### Description

TapGestureRecognizer(setting on Path) does not work on iOS but it works on Android . 

### Steps to Reproduce

1. Create a simple Path and add TapGestureRecognizer 

```
  <Path Data="M 10,100 L 100,100 100,50Z"
      Stroke="Black"
      StrokeThickness="5"
      Aspect="Uniform"
      HorizontalOptions="Start" >
        <Path.GestureRecognizers>
            <TapGestureRecognizer Tapped="TapGestureRecognizer_Tapped"/>
        </Path.GestureRecognizers>
    </Path>

```
2. Click on the path .

### Expected Behavior

TapGestureRecognizer_Tapped event triggers

### Actual Behavior

TapGestureRecognizer_Tapped event not trigger

### Basic Information

Xamarin.Forms : 4.8.0.1687
Xamarin.iOS : 14.4.1.3

Test on iPhone 8 simulator

**Comment by jsuarezruiz** [comment]
Is the same as #10623 and will be fixed by #11419

**Comment by amura11** [comment]
@jsuarezruiz sorry if this is answered somewhere else, I couldn't find it myself, is there any timeline on when 5.0.0 will be released? The workaround mentioned in #10623 didn't work for me and I'm in a bit of a time crunch. If 5.0.0 won't be available in the near future I can look into an alternative approach (maybe just use an image button for now)

**Comment by jsuarezruiz** [comment]
@amura11 In the next weeks. I can let you know when having a specific date.
